config/api/models/File.php

```diff
@@ -59,9 +59,6 @@
         'options' => function (File $file) {
             return $file->panel()->options();
         },
-        'panelIcon' => function (File $file) {
```

Comment:
Probably better to deprecate it and return the full `panel()->image()` information for the time being - that would at least include all previous keys/values of `->icon()`